From 959f6ba2197c631b090ed13053adefbc450fcb51 Mon Sep 17 00:00:00 2001
From: Benjamin Peterson <benjamin@python.org>
Date: Tue, 6 Oct 2015 20:15:30 -0700
Subject: [PATCH] Added tag 1.10.0 for changeset e5218c3f66a2

---
 .hgtags | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.hgtags b/.hgtags
index be68f64..e77cfdd 100644
--- a/.hgtags
+++ b/.hgtags
@@ -16,3 +16,4 @@ cb0ed571b64a7fd87d2062b2f7350b671d842095 1.7.2
 31d9088544784ca0dd3bffc7e14df4f80327016d 1.7.3
 fa79f187d6f288d698f2486358d70df688074a94 1.8.0
 a9b120c9c49734c1bd7a95e7f371fd3bf308f107 1.9.0
+e5218c3f66a2614acb7572204a27e2b508682168 1.10.0
-- 
1.8.3

