From 1a42e8b2ace42b4808510ebb4ad8af8e964fa543 Mon Sep 17 00:00:00 2001
From: Blagovesta Stanoeva <bstanoeva@vmware.com>
Date: Wed, 21 Jun 2017 10:33:31 +0300
Subject: [PATCH 1/4] Modify use_scm_version

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 82e55fc..d0478b1 100644
--- a/setup.py
+++ b/setup.py
@@ -14,7 +14,7 @@ except(IOError, ImportError):
 
 setup(
     name='Automat',
-    use_scm_version=True,
+    use_scm_version={"root": "..", "relative_to": __file__},
     url='https://github.com/glyph/Automat',
     description="""
     Self-service finite-state machines for the programmer on the go.
-- 
2.8.0.windows.1

