From eade7761ef05016947da5684d0b3aa692d877fd4 Mon Sep 17 00:00:00 2001
From: "Md.Borhan Uddin" <uddinm@vmware.com>
Date: Tue, 20 Jun 2017 22:24:52 -0700
Subject: [PATCH] Fix the build issue of python-attrs 17.2.0 on windows

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 68b26e9..8180e97 100644
--- a/setup.py
+++ b/setup.py
@@ -65,7 +65,7 @@ LONG = (
     read("README.rst") + "\n\n" +
     "Release Information\n" +
     "===================\n\n" +
-    re.search("(\d+.\d.\d \(.*?\)\n.*?)\n\n\n----\n\n\n",
+    re.search("(\d+.\d.\d \(.*?\)\r?\n.*?)(\r?\n){3}----(\r?\n){3}",
               read("CHANGELOG.rst"), re.S).group(1) +
     "\n\n`Full changelog " +
     "<{uri}en/stable/changelog.html>`_.\n\n".format(uri=URI) +
-- 
2.8.0.windows.1

