From b13dd6b2cd800c71b738a97b1855d4c414e56745 Mon Sep 17 00:00:00 2001
From: Blagovesta Stanoeva <bstanoeva@vmware.com>
Date: Wed, 21 Jun 2017 14:04:52 +0300
Subject: [PATCH 3/4] Remove setuptools-scm

---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index b2a6e6b..f9c224a 100644
--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,6 @@ except(IOError, ImportError):
 setup(
     name='Automat',
     version='0.6.0',
-    use_scm_version=True,
     url='https://github.com/glyph/Automat',
     description="""
     Self-service finite-state machines for the programmer on the go.
@@ -24,7 +23,6 @@ setup(
     packages=find_packages(exclude=[]),
     package_dir={'automat': 'automat'},
     setup_requires=[
-        'setuptools-scm',
         'm2r',
     ],
     install_requires=[
-- 
2.8.0.windows.1

